File: 1020/1020.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

var num = parseInt(lines.shift());

var ano = Math.floor(num / 365);

num = num % 365;

var mes = Math.floor(num / 30);

var dia = num % 30;

console.log(ano + " ano(s)");
console.log(mes + " mes(es)");
console.log(dia + " dia(s)");

File: 1020/1020.py
num = int(input())

ano = 0
mes = 0
dia = 0

while (num >= 365):
    num -= 365
    ano += 1
    
while (num >= 30):
    num -= 30
    mes += 1
    
while (num >= 1):
    num -= 1
    dia += 1
    
print(f"{ano} ano(s)")
print(f"{mes} mes(es)")
print(f"{dia} dia(s)")